test/e2e/Makefile

```diff
@@ -56,6 +53,10 @@ SKIP_RESOURCE_CLEANUP ?= false
 USE_EXISTING_CLUSTER ?= false
 GINKGO_NOCOLOR ?= false
 
+ginkgo: $(GINKGO)
+$(GINKGO): $(TOOLS_DIR)/go.mod # Build kustomize from tools folder.
+	cd $(TOOLS_DIR) && go build -tags=tools -o $(BIN_DIR)/ginkgo github.com/onsi/ginkgo/ginkgo
```

Comment:
Something is inconsistent here. It checks if `$(GINKGO)` exists, which resolves to `$(TOOLS_BIN_DIR)/ginkgo`, which resolves to `$(TOOLS_DIR)/$(BIN_DIR)/ginkgo`, which resolves to `$(REPO_ROOT)/hack/tools/bin/ginkgo`, but then you are building it to `-o $(BIN_DIR)/ginkgo`, which is just `./bin/ginkgo`. I think you wanted to target the actual `$(GINKGO)`?

It might be easier to just do 

```makefile
go build -o $@ github.com/onsi/ginkgo/ginkgo
```